src/Engine/ProtoCore/AssociativeGraph.cs

```diff
@@ -384,7 +384,7 @@ public static List<AssociativeGraph.GraphNode> GetGraphNodesFromAST(ProtoCore.DS
         public static bool IsGlobalScopeDirty(Executable exe)
         {
             Validity.Assert(exe != null);
-            var graph = exe.instrStreamList[0].dependencyGraph;
+            var graph = exe.GetInstructionStream(0).dependencyGraph;
```

Comment:
GetInstructionStream is minor cleanup work to isolate references to the instruction stream.